WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -776,6 +870,26 @@ private void showPluginRemoveFailedSnackbar() {
                 .show();
     }
 
+    private void showDomainCreditsCheckProgressDialog() {
+        if (mCheckingDomainCreditsProgressDialog == null) {
+            mCheckingDomainCreditsProgressDialog = new ProgressDialog(this);
+            mCheckingDomainCreditsProgressDialog.setCancelable(false);
+            mCheckingDomainCreditsProgressDialog.setIndeterminate(true);
+
+            mCheckingDomainCreditsProgressDialog
+                    .setMessage(getString(R.string.plugin_check_domain_credits_progress_dialog_message));
+        }
+        if (!mCheckingDomainCreditsProgressDialog.isShowing()) {
+            mCheckingDomainCreditsProgressDialog.show();
```

Comment:
The dialog leaks on configuration change we should dismiss it manually.

Reply:
Good catch!